circleci-ruby/2.6.2-stretch-pgclient11.1-node10.15.1-browsers/Dockerfile

```diff
@@ -0,0 +1,182 @@
+FROM circleci/ruby:2.6.2-stretch
+
+USER root
+
+
+#### PG CLIENT
+
+ENV PG_VERSION 11
+
+RUN mkdir ~/.gnupg
+RUN chmod 600 ~/.gnupg
+RUN echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf
+
```

Comment:
Disable ipv6 support for gnupg, workaround as described here:
https://github.com/inversepath/usbarmory-debian-base_image/issues/9

Reply:
Thanks for the link and context.